### PR TITLE
Allow to fix seed when querying blackbox

### PR DESCRIPTION
**Pull request by mseeger**
*Issue #, if available:*

*Description of changes:*
Allows seed for blackbox queries to be fixed, so that these queries become deterministic. This is needed if a blackbox is used for testing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mseeger** [comment]
OK, fixed_seed -> seed.

I also made sure that the same (random) seed is used for a trial, even if it is resumed.

File: benchmarks/launch_utils.py [hunk 1]
```diff
@@ -155,6 +155,10 @@ def parse_args(allow_lists_as_values=True):
     parser.add_argument('--blackbox_repo_s3_root', type=str,
                         help='S3 root directory for blackbox repository. '
                              'Defaults to default bucket of session')
+    parser.add_argument('--blackbox_fixed_seed', type=int,
```
Comment (geoalgo) [inline benchmarks/launch_utils.py:158]: Could you rename this one to `blackbox_seed` too for consistency?

**Review by geoalgo — COMMENTED** [review]
Can you also rename `blackbox_fixed_seed` to `blackbox_seed` for consistency?

File: benchmarks/launch_hpo.py [hunk 2]
```diff
@@ -278,6 +278,7 @@
                     'time_this_resource_attr': benchmark.get(
                         'time_this_resource_attr'),
                     'max_resource_attr': benchmark.get('max_resource_attr'),
+                    'seed': params.get('blackbox_fixed_seed'),
```
[the remark this answers is not in the document]
Comment (mseeger) [reply benchmarks/launch_hpo.py]: Done

**Review by geoalgo — APPROVED** [review]

**Pull request merged**